docs/UserGuide.md

```diff
@@ -147,6 +147,7 @@ The only possible values for a review status are:
 - `"wikidata"` - The mismatching information is on Wikidata.
```

Comment:
Not caused by this PR, but I find this line highly confusing. "mismatching" in this context mean "correct" or "incorrect"?